frontend/src/App.vue

```diff
@@ -1,7 +1,18 @@
-<script setup lang="ts">
+<script lang="ts">
+import { defineComponent } from 'vue'
+import Avatar from './components/Avatar/Avatar.vue';
 
+export default defineComponent({
+  name: 'App',
+  components: {
+    Avatar
+  }
+})
 </script>
 
 <template>
-    <p class="m-3">HELLO WORLD</p>
-</template>
\ No newline at end of file
+    <div>
+      <h1>Avatar Component</h1>
```

Comment:
Lets not put the avatar directly into App.vue as this is not where it should go. When creating a component, only call it where it should be called. (fine for testing in dev but not when suitable for a pr)

Reply:
I have deleted the h1 heading for clarity. I will keep the Avatar component in App.vue for now until we have a page where we can put the Avatar component. Thank you for the note!

Reply:
Ah ok, thank you! Removed from App.vue.